study-builder/tenants/prion/loginPage.html

```diff
@@ -919,6 +919,9 @@
             'lock.fallback': 'We\'re sorry, something went wrong when attempting to log in.',
             'lock.invalid_email_password': 'Wrong email or password.',
             'lock.network': 'We could not reach the server. Please check your connection and try again.'
+        },
+        passwordStrength: {
+            lengthAtLeast: 'At least %d characters in length'
```

Comment:
I think there are other text snippets too. Take a look at `password_reset.html` file for the `testboston` tenant in study-builder.